src/MatchMaker.ts

```diff
@@ -139,6 +139,10 @@ export class MatchMaker {
     // if couldn't join a room by its id, let's try to create a new one
     if (!roomId && hasHandler) {
       roomId = this.create(roomToJoin, clientOptions);
+
+      if (!roomId) {
+        throw new Error(`Failed to auto-create room "${roomToJoin}" during join request using options "${JSON.stringify(clientOptions)}"`);
```

Comment:
Travis is not passing because of this line:

```
ERROR: /home/travis/build/gamestdio/colyseus/src/MatchMaker.ts[144, 1]: Exceeds maximum line length of 120
```